packages/cli/src/cli/github/make.cog.pr.ts

```diff
@@ -127,4 +103,44 @@ export class MakeCogGithub extends Github {
 
     return tileSet;
   }
+
+  /**
+   * Prepare and create pull request for the aerial tileset config
+   */
+  async updateVectorTileSet(config: string, layer: ConfigLayer): Promise<void> {
```

Comment:
`config` is pretty generic, what config is this related to?

this needs either a doc or a better name or both?

Reply:
I think I will change this to `filename`